pyqtgraph/graphicsItems/TargetItem.py

```diff
@@ -459,7 +459,7 @@ def mouseClickEvent(self, ev):
 
     def mouseDragEvent(self, ev):
         targetItem = self.parentItem()
-        if not targetItem.movable or int(ev.button() & QtCore.Qt.LeftButton) == 0:
+        if not targetItem.movable or int(ev.button() & QtCore.Qt.MouseButton.LeftButton) == 0:
```

Comment:
Another cast to int that doesn't work in PyQt6

Reply:
Nice catch!